<!--
  DIGMA-Action Camera
  Attention: video_device=...
  Можно запускать с аргументами:
    roslaunch yyctl cam.launch device:=/dev/video0
-->

<launch> 
  <arg name="device" default="/dev/video0"/>

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" > 
    <param name="video_device" value="$(arg device)" />
<!--
    <param name="pixel_format" value="yuyv"/>
-->
    <param name="image_width" value="800" />
    <param name="image_height" value="600" />

    <param name="camera_frame_id" value="usb_cam" />
    <!--
    <param name="io_method" value="mmap"/>
    -->
    <param name="autoexposure" value="true" />
    <param name="auto_whitebalance" value="true" />
    <param name="auto_focus" value="true" />
    <param name="auto_brigthness" value="true" />
  </node>

<!--
<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen"> 
    <remap from="image" to="/usb_cam/image_raw"/> 
    <param name="autosize" value="true" /> 
  </node>
-->
</launch>
